test/quantization/fx/test_quantize_fx.py

```diff
@@ -2423,6 +2423,120 @@ def forward(self, x):
             mp(torch.rand(4, 4, 4, 4))
             mc = convert_fx(mp)
 
+    def _check_not_observed(self, model: nn.Module, node_name_list: List[str]):
+        # check that no obsersvers have the named nodes as an input
+        for node in model.graph.nodes:
+            if hasattr(model, node.name) and isinstance(
+                getattr(model, node.name), torch.ao.quantization.observer.ObserverBase
+            ):
+                for arg in node.args:
+                    self.assertTrue(arg.name not in node_name_list)
+
+    class _non_referece_test_model(nn.Module):
+        def __init__(self, func, lin_in, lin_out):
+            super().__init__()
+            self.conv1 = nn.Conv2d(3, 6, 5)
+            self.pool = nn.MaxPool2d(2, 2)
+            self.lin = nn.Linear(lin_in, lin_out)
+            self.func = func
+
+        def forward(self, x, y, z):
+            x = self.pool(F.relu(self.conv1(x)))
+            x = torch.flatten(x, 1)
+            x = self.func(x, y, z)
+            x = self.lin(x)
+            return x
+
+    def _test_dtype_propagation(self, model, *args):
```

Comment:
I guess it is trying to verify that `y` and `z` are not observed, since we know the these arguments are not Tensors based on non_observable_arg_list, but would be better if we can directly document these things

Reply:
Ya i'll add a comment